buildbot/master.cfg

```diff
@@ -249,7 +242,7 @@ def make_dolphin_win_build(build_type, arch, mode="normal"):
                                          "HASH": WithProperties("%s", "revision"),
                                          "AUTHOR": WithProperties("%s", "author"),
                                          "DESCRIPTION": WithProperties("%s", "description"),
-                                         "TARGET_SYSTEM": "Windows x86" if arch == "x86" else "Windows x64",
+                                         "TARGET_SYSTEM": "Windows arm64" if arch == "arm64" else "Windows x64",
```

Comment:
Maybe a bit pedantic but should arm be capitalized here? Microsoft appears to use either "ARM64" or "Arm64" in documentation.

Reply:
maybe? tbh I'm not sure if this is user-facing field or not.

Reply:
shrug i'll just capitalize it in some more places

Reply:
Oh yeah I was just thinking in user facing text. If this doesn't appear on the website feel free to ignore.